lib/resources/example_json/dstu2_examples_schedule.rb

```diff
@@ -2,74 +2,124 @@ module Cerner
   module Resources
 
     DSTU2_SCHEDULE_ENTRY ||= {
-        "resourceType": "Schedule",
-        "id": "166627791-4048128-9331486-0",
-        "text": {
-            "status": "generated",
-            "div": "&lt;div&gt;&lt;p&gt;&lt;b&gt;Schedule&lt;/b&gt;&lt;/p&gt;&lt;p&gt;&lt;b&gt;Type&lt;/b&gt;: Unscheduled walkins&lt;/p&gt;&lt;p&gt;&lt;b&gt;Practitioner&lt;/b&gt;: Hall, Mark&lt;/p&gt;&lt;p&gt;&lt;b&gt;Location&lt;/b&gt;: Baseline East&lt;/p&gt;&lt;/div&gt;"
-        },
-        "extension": [
-            {
-                "url": "https://fhir-ehr.cerner.com/dstu2/StructureDefinition/scheduling-location",
-                "valueReference": {
-                    "reference": "Location/4048128",
-                    "display": "Baseline East"
-                }
-            }
-        ],
-        "type": [
+      "resourceType": "Schedule",
+      "id": "21265426-633867-3121665-0",
+      "text": {
+        "status": "generated",
+        "div": "&lt;div&gt;&lt;p>&lt;b>Schedule&lt;/b>&lt;/p>&lt;p>&lt;b>Type&lt;/b>: Same Day&lt;/p>&lt;p>&lt;b>Practitioner&lt;/b>: Song, River&lt;/p>&lt;p>&lt;b>Location&lt;/b>: Baseline West&lt;/p>&lt;/div>"
```

Comment:
Some closing brackets are html-escaped here and some are not - doesn't matter when rendered but would be nice to have them consistent

Reply:
Addressed with f54f352a457572e639496cf1f96efa62816cd041